admin-dev/filemanager/upload.php

```diff
@@ -38,14 +38,14 @@
     $path = fix_dirname($path).'/';
 }
 
-if (!empty($_FILES) && isset($_FILES['file']) && $_FILES['file']['size']) {
+if (!empty($_FILES['file']) && !empty($_FILES['file']['name']) && !empty($_FILES['file']['size'])
+        && is_uploaded_file($_FILES['file']['tmp_name'])) {
     $info = pathinfo($_FILES['file']['name']);
-    if (isset($info['extension'])
-            && in_array(fix_strtolower($info['extension']), $ext)
-            && in_array(mime_content_type($_FILES['file']['tmp_name']), $mime)
-    ) {
-        $tempFile = $_FILES['file']['tmp_name'];
-
+    $tempFile = sys_get_temp_dir() . '/ps__filemanager__' . time() . '__'
```

Comment:
seems really overkill when php provides: https://www.php.net/manual/en/function.tmpfile.php
And Symfony also provides a way to do ti :thinking:

Reply:
How can `tmpfile()` method be used with `move_uploaded_file`?